mobile/src/main/java/ch/epfl/sdp/mobile/ui/speech_recognition/ChessDictionary.kt

```diff
@@ -0,0 +1,15 @@
+package ch.epfl.sdp.mobile.ui.speech_recognition
+
+/**
+ * Interface representing the vocabulary of chess move commands
+ * @property chessPieces mapping of chess pieces keywords to their homophones
+ * @property numbers mapping of chess number keywords to their homophones
+ * @property letters mapping of chess letter keywords to their homophones
+ * @property placements all possible combinations of chess board placement
+ */
+interface ChessDictionary {
+  val chessPieces: Map<String, List<String>>
+  val numbers: Map<String, List<String>>
+  val letters: Map<String, List<String>>
+  val placements: List<String>
```

Comment:
Question : is the ordering important ? Or should we use some `Set` instead ?

Reply:
No, it was just for convenience, I changed it to Set. Now, the implementation transforms the computed list to a set after doing the mappings on the keys.